tasks/modules.py

```diff
@@ -310,6 +310,10 @@ def dependency_path(self, agent_version):
     # Test files
     Path("./test/integration/serverless/recorder-extension"),
     Path("./test/integration/serverless/src"),
+    # TODO: Remove comp/core/log and pkg/util/tagger once otel agent dependencies are fixed
+    Path("./comp/core/log"),
+    Path("./pkg/util/tagger"),
```

Comment:
the older version of datadogexporter or datadogconnector that was pinned in the codebase; this will be fixed after the k8s dependencies is merged bumping to 0.111.0